Celeste.Mod.mm/Mod/Core/CoreModuleSettings.cs

```diff
@@ -308,6 +308,56 @@ public bool ColorizedLogging {
         [SettingIgnore]
         public int DebugRCPort { get; set; } = 32270;
 
+        public bool PhotosensitiveMode {
+            get => Settings.Instance.DisableFlashes;
+            set => Settings.Instance.DisableFlashes = value;
+        }
+        
+        [SettingIgnore]
+        public bool PhotosensitivityDistortOverride { get; set; } = false;
+
+        [SettingIgnore]
+        [YamlIgnore]
+        public bool AllowDistort {
+            get => !Settings.Instance.DisableFlashes || PhotosensitivityDistortOverride; 
+        }
```

Comment:
```suggestion
        public bool AllowDistort => !Settings.Instance.DisableFlashes || PhotosensitivityDistortOverride; 
```
Same applies in a bunch of places.

Reply:
Resolved in commit [6f4ef7f](https://github.com/EverestAPI/Everest/pull/836/commits/6f4ef7fd93c9b4f858749dc54c82c2e1e52a41e8)